src/hotspot/share/gc/g1/g1SegmentedArray.hpp

```diff
@@ -215,8 +215,8 @@ class G1SegmentedArray : public FreeListConfig  {
   G1SegmentedArrayFreeList* _free_segment_list;   // The global free segment list to preferentially
                                                   // get new segments from.
 
-  volatile uint _num_available_slots; // Number of slots available in all segments (allocated + free + pending + not yet used).
-  volatile uint _num_allocated_slots; // Number of total slots allocated and in use.
+  volatile uint _num_available_slots; // Number of slots available in all segments (allocated + not yet used).
```

Comment:
I wonder if `_num_total_slots` is better; "available" seems to suggest a slot that can be "used" when a request comes in.